src/Evolution/Systems/GeneralizedHarmonic/Tags.hpp

```diff
@@ -166,6 +166,52 @@ struct EvolvedFieldsFromCharacteristicFields : db::SimpleTag {
     return "EvolvedFieldsFromCharacteristicFields";
   }
 };
+
+// @{
+/*!
+ * \brief Tags corresponding to various constraints of the generalized
+ * harmonic system, and their diagnostically useful combinations.
+ * \details For details on how these are defined and computed, see
+ * `GaugeConstraintCompute`, `FConstraintCompute`, `TwoIndexConstraintCompute`,
+ * `ThreeIndexConstraintCompute`, `FourIndexConstraintCompute`, and
+ * `ConstraintEnergyCompute` respectively
+ */
+template <size_t SpatialDim, typename Frame>
+struct GaugeConstraint : db::SimpleTag {
+  using type = tnsr::a<DataVector, SpatialDim, Frame>;
+  static std::string name() noexcept { return "GaugeConstraint"; }
+};
+//! \copydoc GaugeConstraint
```

Comment:
Let's not introduce another Doxygen syntax and use `/// \copydoc GaugeConstraint` (triple slash instead of double slash + exclamation point) :)

Reply:
changed